src/app/home/pages/PlansPage/components/PlanStatusIcon.tsx

```diff
@@ -82,6 +84,12 @@ const PlanStatusIcon: React.FunctionComponent<IProps> = ({ plan }) => {
         <ResourcesAlmostEmptyIcon />
       </span>
     );
+  } else if (latestAction === 'rollback' && hasSucceededCondition) {
```

Comment:
@ibolton336 you're gonna kill me, but I double checked and there is already a `hasSucceededRollback` on the `plan?.PlanStatus` that you can use instead of this line, just like `hasSucceededStage` and `hasSuccededCutover` above. See https://github.com/konveyor/mig-ui/blob/master/src/app/plan/duck/selectors.ts#L189-L198

I think that's actually more accurate than this because `hasSucceededCondition` is based on conditions on the plan and not the migration (I'm not sure if the plan gets a success condition when a rollback succeeds but the rollback's migmigration definitely does)

Reply:
Oops, I see that hasSucceededRollback checks that the latest action is a rollback.

Reply:
yeah it's not the best naming :)